plugins/scanners/scancode/src/main/kotlin/ScanCodeResultModelMapper.kt

```diff
@@ -85,12 +85,20 @@ fun ScanCodeResult.toScanSummary(preferFileLicense: Boolean = false): ScanSummar
 
     filesOfTypeFile.forEach { file ->
         val licensesWithoutReferences = file.licenses.filter {
-            it !is LicenseEntry.Version3 || it.fromFile == null
+            // Determine `fromFile`, which we have starting from version 3
+            var fromFile: String? = null
+            if (it is LicenseEntry.Version3) {
```

Comment:
commit-msg: Please add some short explanation about the commit. IMO parts of the PR description could be re-used.

Reply:
The commit message requires to adhere to 75 chars line length limit in the body, and an empty line above the sign off.

Reply:
You're right, thank you. I fixed the message 👍